testing-overhead-jmh/src/jmh/java/io/opentelemetry/javaagent/benchmark/servlet/ServletWithAgentDisabledBenchmark.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.benchmark.servlet;
+
+import org.openjdk.jmh.annotations.Fork;
+
+@Fork(jvmArgsAppend = "-Dotel.javaagent.enabled=false")
```

Comment:
Just an idea: we could also run a benchmark with `-Dotel.javaagent.experimental.use-noop-api=true` to see how much overhead is added when no SDK at all is used.

Reply:
added 👍